### PR TITLE
Base Typography component defined

### PR DESCRIPTION
**Pull request by pallavikadam-srijan**
Added Base typography atom

### CODE REVIEW

File: stories/Atom/Base-typography/Link/link.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './link.scss';
+
+export const Link = ({ label }) => {
+  return (
+    <a href="">
```
Comment (surbhi90) [inline stories/Atom/Base-typography/Link/link.jsx:6]: add # in href

File: stories/Atom/Base-typography/Mark/mark.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@import "../../../assets/scss/mixins";
+@import "../../../assets/scss/breakpoints";
+@import "../../../assets/scss/variables";
+
+mark {
+    @extend %paragragh-font-20;
+    background: $color-light-yellow;
+    padding: 2px;
+    // font-size: 14px;
+    // line-height: 19px;
```
Comment (surbhi90) [inline stories/Atom/Base-typography/Mark/mark.scss:10]: @pallavikadam-srijan remove the commented code

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Atom/Base-typography/Link/link.jsx (hunk 1 above)
Comment (pallavikadam-srijan) [reply stories/Atom/Base-typography/Link/link.jsx]: Done

File: stories/Atom/Base-typography/Mark/mark.scss (hunk 2 above)
Comment (pallavikadam-srijan) [reply stories/Atom/Base-typography/Mark/mark.scss]: Done

**Pull request merged**